modules/monitoring/manifests/init.pp

```diff
@@ -31,6 +31,10 @@
     include monitoring::uptime_collector
   }
 
+  if $::aws_environment == 'production' {
+    include monitoring::gcloud
+  }
+
```

Comment:
This conditional is unnecessary since you have `enabled` defaulting to false, so the presence/absence of the alert is already neatly controlled by setting `enabled` in Hiera in production.

Reply:
My reasoning for doing it this way was to only allow the gcloud sdk to be installed on the machines that really need it, and reducing the size of the puppet catalog.  By having it here, without the conditional, it will install on all monitoring boxes.  I think an alternative could be moving the install of the packages to within the check, which would mean it gets captured by the $enable.

This gcloud sdk is also installed here: modules/govuk_ci/manifests/agent/gcloud.pp.  There is certainly some improvement that can be done around this and I shall ponder it over night.

Reply:
Ohh sorry, I misread this in a very silly way (as if it said `monitoring::checks::mirror` rather than `monitoring::gcloud`) and mixed it up with the earlier comment about `enabled`. So I was talking complete nonsense here 🙈 Let's put it down to a year of isolation and hope I'm not just losing my marbles 😅 

No big deal then but I wonder if it's worth the small cost of installing gcloud in all the environments if only to keep non-prod similar to prod so there's a chance of catching any future breakage? I'm fine with whatever you think best on this.

Reply:
I agree with the small cost of installing on all environments.  Cheers for the feedback.